worlds/dark_souls_3/__init__.py

```diff
@@ -0,0 +1,280 @@
+        irithyll_of_the_boreal_valley_region.exits.append(Entrance(self.player, "Goto Irithyll dungeon",
+                                                                   irithyll_of_the_boreal_valley_region))
+        irithyll_of_the_boreal_valley_region.exits.append(Entrance(self.player, "Goto Anor Londo",
+                                                                   irithyll_of_the_boreal_valley_region))
+        self.world.get_entrance("Goto Irithyll dungeon", self.player).connect(irithyll_dungeon_region)
+        self.world.get_entrance("Goto Anor Londo", self.player).connect(anor_londo_region)
+        irithyll_dungeon_region.exits.append(Entrance(self.player, "Goto Archdragon peak", irithyll_dungeon_region))
+        irithyll_dungeon_region.exits.append(Entrance(self.player, "Goto Profaned capital", irithyll_dungeon_region))
+        self.world.get_entrance("Goto Archdragon peak", self.player).connect(archdragon_peak_region)
+        self.world.get_entrance("Goto Profaned capital", self.player).connect(profaned_capital_region)
+        lothric_castle_region.exits.append(Entrance(self.player, "Goto Consumed King Garden", lothric_castle_region))
+        lothric_castle_region.exits.append(Entrance(self.player, "Goto Grand Archives", lothric_castle_region))
+        self.world.get_entrance("Goto Consumed King Garden", self.player).connect(consumed_king_garden_region)
+        self.world.get_entrance("Goto Grand Archives", self.player).connect(grand_archives_region)
+        consumed_king_garden_region.exits.append(Entrance(self.player, "Goto Untended Graves",
+                                                          consumed_king_garden_region))
+        self.world.get_entrance("Goto Untended Graves", self.player).connect(untended_graves_region)
+
+    # For each region, add the associated locations retrieved from the corresponding location_table
+    def create_region(self, region_name, location_table) -> Region:
+        new_region = Region(region_name, RegionType.Generic, region_name, self.player)
+        if location_table:
+            for name, address in location_table.items():
+                location = Location(self.player, name, self.location_name_to_id[name], new_region)
+                new_region.locations.append(location)
+        self.world.regions.append(new_region)
+        return new_region
+
+    def create_items(self):
+        for name, address in self.item_name_to_id.items():
+            # Specific items will be included in the item pool under certain conditions. See generate_basic
+            if name != "Basin of Vows":
+                self.world.itempool += [self.create_item(name)]
+
+    def generate_early(self):
+        pass
+
+    def set_rules(self) -> None:
+
+        # Define the access rules to the entrances
+        set_rule(
+            self.world.get_entrance("Goto Bell Tower", self.player),
+            lambda state: state.has("Mortician's Ashes", self.player))
```

Comment:
In core code we format calls>120chars like so:
```python
        set_rule(self.world.get_entrance("Goto Bell Tower", self.player),
                 lambda state: state.has("Mortician's Ashes", self.player))
```
I don't have a strong opinion though, so if you think your version is better readable, I don't mind, and simply mark this as resolved.

Reply:
Let's favor code consistency! It will make it more readable for all of us.
Fixed